releng/org.obeonetwork.dsl.uml2.core.target/umlcoredesigner-capella-1.2.tpd

```diff
@@ -1,31 +1,44 @@
 target "UML Core Designer Neon for Capella"
```

Comment:
Precise the X in '1.2.X' in the tpd file name, as this is dedicated to an exact capella version